config/reports.json

```diff
@@ -6,10 +6,12 @@
 		"loading-speed",
 		"search-engine-optimization",
 		"page-weight",
-		"chrome-ux-report"
+		"chrome-ux-report",
+		"progressive-web-apps"
```

Comment:
This ordering determines where the report appears in the UI. We can move this up to a more prominent spot, like after loading-speed.

Reply:
Addressed in https://github.com/HTTPArchive/httparchive.org/pull/51/commits/f318a28d68f1419f902e0918cf26f446a407ffc2.